apps/mobile/pushtracker/src/app/services/bluetooth.service.ts

```diff
@@ -230,21 +244,32 @@ export class BluetoothService extends Observable {
     this.initialized = true;
   }
 
-  async advertise(): Promise<any> {
+  async advertise(): Promise<boolean> {
```

Comment:
Since we no longer return a boolean

```suggestion
  async advertise() {
```